sqlx-macros/src/query/input.rs

```diff
@@ -51,8 +53,11 @@ impl Parse for QueryMacroInput {
             let _ = input.parse::<syn::token::Eq>()?;
 
             if key == "source" {
-                let lit_str = input.parse::<LitStr>()?;
-                query_src = Some((QuerySrc::String(lit_str.value()), lit_str.span()));
+                let query_str = Punctuated::<LitStr, Token![+]>::parse_separated_nonempty(input)?
+                    .iter()
+                    .map(LitStr::value)
+                    .collect();
+                query_src = Some((QuerySrc::String(query_str), input.span()));
```

Comment:
You should take `input.span()` first since after parsing out the collection of strings it'll be pointing to the next part of the input.